packages/test-utils/src/reporter.ts

```diff
@@ -900,12 +904,18 @@ class ReplayReporter {
       output.push(`\n🚀 Successfully uploaded ${uploads.length} recordings:\n`);
```

Comment:
It'll first log "successfully uploaded..." and then later in output we'll say the recording failed to upload. That feels a bit odd if my understanding is right.